packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -204,6 +247,14 @@ function APIProvider({ children }) {
         path,
         data: getStorySaveData(story),
         method: 'POST',
+      }).then((data) => {
+        const { _embedded: embedded = {} } = data;
+
+        if ('wp:featuredmedia' in embedded) {
+          data.featured_media_url = embedded['wp:featuredmedia'][0].source_url;
```

Comment:
Same thing here about camel cases.

Also wondering why we don't just update `featuredMedia` and instead have this separate field 🤔 

Seems redundant.

Reply:
https://github.com/google/web-stories-wp/pull/8990#discussion_r708094816

Reply:
Right, that's for the camel case. But the `featured_media_url` vs. `featuredMedia` part seems like a bug (not in this PR, but also on `main`). It's a bit confusing, to say the least.

Reply:
What I mean is, we use `featuredMedia` everywhere, for example in `useLoadStory`.

But in `useSaveStory` we update `featuredMediaUrl` instead of `featuredMedia`

Reply:
Is this what you are referring to?
https://github.com/google/web-stories-wp/pull/8990/commits/f65324a6433be2fe281e797ad81be3b071f60efb